test/explorer/query_test.exs

```diff
@@ -5,11 +5,11 @@ defmodule Explorer.QueryTest do
   require Explorer.DataFrame, as: DF
   doctest Explorer.Query
 
-  test "allows accessing the dataframe" do
-    assert DF.new(a: [1, 2, 3])
-           |> DF.filter(df()["a"] < ^if(true, do: 3, else: 1))
-           |> DF.to_columns(atom_keys: true) == %{a: [1, 2]}
-  end
+  # test "allows accessing the dataframe" do
+  #   assert DF.new(a: [1, 2, 3])
+  #          |> DF.filter(df()["a"] < ^if(true, do: 3, else: 1))
+  #          |> DF.to_columns(atom_keys: true) == %{a: [1, 2]}
+  # end
```

Comment:
This test is no longer passing. It was added 2 weeks ago as part of this:

* https://github.com/elixir-explorer/explorer/issues/978#issuecomment-2329414215

Since I removed the use `Explorer.Query.df_var/0`, I'm not sure we need this test. The only reason I didn't remove it entirely was because of the docs for `col` and `across`, which I'm not sure what to do about.

Reply:
[7ac1ada](https://github.com/elixir-explorer/explorer/pull/989/commits/7ac1adaef2169e7d6478f721efa62ebd0f1a0d8c)